ffv1.md

```diff
@@ -186,19 +188,19 @@ a = b, a += b, a -= b, a *= b
 
 `get_bits( i )` is the action to read the next `i` bits in the bitstream, from most significant bit to least significant bit, and to return the corresponding value. The pointer is increased by `i`.
 
-# General Description
+# Sample Encoding
 
-Samples within a plane are coded in raster scan order (left->right, top->bottom). Each sample is predicted by the median predictor from samples in the same plane and the difference is stored see [Coding of the Sample Difference](#coding-of-the-sample-difference).
+For each `Slice` (as described in [the section on `Slices`](#slice)) of a `Frame`, the samples are coded in an order determined by the `colorspace_type` (see [the section on `colorspace_type`](#colorspacetype)). Each sample is predicted by the median predictor as described in [the section of the Median Predictor](#median-predictor) from other samples within the same plane and the difference is stored using the method described in [Coding of the Sample Difference](#coding-of-the-sample-difference).
 
```

Comment:
scan order (left->right, top->bottom) is no more indicated (`colorspace_type` does not indicate such order, it is common to all `colorspace_type`), it should be indicated somewhere else if not here.

Reply:
I was concerned that `scan order (left->right, top->bottom)` was misleading since the samples are ordered like:
- RGB is left->right, first plane->last plane, top->bottom
- YCbCr is first plane->last plane, left->right, top->bottom

Reply:
if we go deep in details, the order is:

- RGB is for each line (top->bottom), list planes (Y, U, V), and for each plan list samples (left->right)
- YCbCr is for each plane (Y, Cb, Cr), list lines (top->bottom), and for each line list samples (left->right)

I understand the need to remove it from here, but I don't find elsewhere that lines listing is top to bottom and sample listing is left to right, so IMO it is needed to indicate it somewhere else before removing the same information from this line.

Reply:
updated, I moved a description of the order to the beginning of the section of Color Spaces